docs/src/man/joins1.md

```diff
@@ -0,0 +1,609 @@
+# Joining DataFrames
+
+"I have two different data sets in Julia but I am not sure how to use them. What 
+criteria should I consider? What are the different ways I can join these data sets?"
+
+Sound familiar? You may have come across this question plenty of times on online discussion
+forums. Working with one dataframe is fairly straightforward but things become challenging when
+we have data spread across two or more data frames. This is where the concept of `joins` come in.
```

Comment:
Here (and below) - this is not code, but concept, so better use highlight not code block to signal this
```suggestion
we have data spread across two or more data frames. This is where the concept of *joins* come in.
```